x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestsDataLoader.java

```diff
@@ -770,10 +776,23 @@ private static void loadCsvData(RestClient client, String indexName, URL resourc
         }
     }
 
-    private static String quoteIfNecessary(String value) {
-        boolean isQuoted = (value.startsWith("\"") && value.endsWith("\"")) || (value.startsWith("{") && value.endsWith("}"));
-        boolean isNumeric = value.matches(NUMERIC_REGEX);
-        return isQuoted || isNumeric ? value : "\"" + value + "\"";
+    private static final Pattern DATE_RANGE_PATTERN = Pattern.compile("([0-9\\-Z:]+)\\.\\.([0-9\\-Z:]+)");
+
+    private static String toJson(String type, String value) {
```

Comment:
I thought we were parsing these and then converting them to json. We aren't. We're just slinging strings around. So I did some light parsing.